### PR TITLE
logging on the swarm with elk

### PR DESCRIPTION
**Pull request by generalhenry**
Creates a swarm, boots up the logger and gets ELK up and running centrally.

note: The IP  (192.168.99.120) in the compose currently need to be updated manually

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -17,28 +19,17 @@ dockerEvents.on('disconnect', function() {
...
-  .listen(12201, function() {
+gelfServer.listen(12201, function() {
     console.log('server started');
-
-    exitOnSignal('SIGINT');
-    exitOnSignal('SIGTERM');
-
-    function exitOnSignal(signal) {
-      process.on(signal, function () {
-        console.log('stopping server...');
```
Comment (subfuzion) [inline server.js]: I would have expected something in this block you removed for graceful server shutdown upon a stop signal to the process, like the following.

```js
gelfServer.close(() => {
          log('server stopped');
          process.exit();
        });
```

**Comment by subfuzion** [comment]
This looks good, Henry! :+1:  I can see the connection info for kibana in the docker-compose file, but maybe you also update the readme with the details as well. Thanks!

File: server.js (hunk 1 above)
Comment (generalhenry) [inline server.js]: To do it properly I'd need to also shut down the docker event client.  For now I'm keeping things down to the core concerns of listening and talking.

**Comment by subfuzion** [comment]
I know you're just prototyping right now, but maybe it will make sense to rename the project to amp-logproxy or amp-logshim, or something to that effect, as we get closer to the point where we can use this.

File: server.js (hunk 1 above)
Comment (subfuzion) [inline server.js]: Ok. Sounds good. I guess you'll just be waiting for the client stream to finish transferring data before exiting the process.

**Pull request merged**